### PR TITLE
Update submodule

### PR DESCRIPTION
**Pull request by trask**

### CODE REVIEW

File: agent/agent-bootstrap/build.gradle.kts [hunk 1]
```diff
@@ -10,6 +10,7 @@ dependencies {
     exclude("org.slf4j", "slf4j-simple")
   }
   implementation("io.opentelemetry.instrumentation:opentelemetry-instrumentation-api")
+  implementation("io.opentelemetry.instrumentation:opentelemetry-instrumentation-api-semconv")
```
Comment (heyams) [inline agent/agent-bootstrap/build.gradle.kts:13]: what is semconv? please explain

**Review by heyams — APPROVED** [review]

File: agent/agent-bootstrap/build.gradle.kts (hunk 1 above)
Comment (trask) [reply agent/agent-bootstrap/build.gradle.kts:13]: `instrumentation-api` was recently split out into 2 artifacts: `instrumentation-api` and `instrumentation-api-semconv`

Comment (heyams) [reply agent/agent-bootstrap/build.gradle.kts:13]: what does semconv stand for?

Comment (trask) [reply agent/agent-bootstrap/build.gradle.kts:13]: semantic conventions

**Pull request merged**